gunrock/app/app.cu

```diff
@@ -0,0 +1,15 @@
+#include <gunrock/app/hits/hits_app.cuh>
+#include <gunrock/app/sm/sm_app.cuh>
+
+#include <tuple>
+
+template <typename... Ts>
+auto instantiate() {
+    static auto func = std::tuple_cat(std::make_tuple(
+    	t_sm<Ts,Ts>
+        )...);
+
+    return &func;
+}
+
+template auto instantiate<int>();
```

Comment:
@crozhon can you help me understand what this is for?

Reply:
Mistake - fixed https://github.com/gunrock/gunrock/commit/b37a2eca2d361c4aa9688da18f798a9ec2aeca9d